components/browser/menu/src/main/java/mozilla/components/browser/menu/view/StickyItemLayoutManager.kt

```diff
@@ -469,6 +470,7 @@ abstract class StickyItemsLinearLayoutManager<T> constructor(
 /**
  * Save / restore existing [RecyclerView] state and scrolling position and offset.
  */
+@SuppressLint("ParcelCreator")
```

Comment:
Oh, this is surprising. I assume this is fixed in a newer version of Android lint? 🤔 Bit annoying to have those. :)

Reply:
Locally, on Android Studio this is not needed, just for CI. Not sure why.